### PR TITLE
feat: add Keyring management for icq-relayer 

### LINKED ISSUE #1656: Keyring management for icq-relayer
## Summary

icq-relayer, prior to v1 used the lens library to handle key management

Since removing the deprecated lens, we need to add in keyring management ourselves. Currently, the config file references a seed file. This is useful in cases where we can store the seed in a proper secrets management solution and mount this on container start, but otherwise this is far from an ideal solution.

As such, we need to add the ability to hook into and leverage the existing Cosmos keyring logic, but also maintain the ability to choose to use the external seed phrase where this is relevant and safe to do so.

Note: this should be built on and PR'd to the `icq/v1.0.x` branch, not `main`.

### PR DESCRIPTION
**Pull request by hoank101**
## 1. Summary
Fixes #1656

## 2.Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## 5. Checklist

- [ ] Does the Readme need to be updated?

### CODE REVIEW

**Comment by hoank101** [comment]
hi @joe-bowman @tropicaldog can you review PR?

**Review by tropicaldog — APPROVED** [review]
LGTM sir @hoank101. Could you include some guide on how the new keyring management work?

**Comment by hoank101** [comment]
> LGTM sir @hoank101. Could you include some guide on how the new keyring management work?

thank you for feedback, I updated

File: icq-relayer/cmd/root.go [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+	"github.com/cosmos/cosmos-sdk/client/config"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/keys"
+	"github.com/cosmos/cosmos-sdk/server"
+	svrcmd "github.com/cosmos/cosmos-sdk/server/cmd"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/quicksilver-zone/quicksilver/app"
+	cmdcfg "github.com/quicksilver-zone/quicksilver/cmd/config"
+	"github.com/spf13/viper"
+	tmcfg "github.com/tendermint/tendermint/config"
```
Comment (odeke-em) [inline icq-relayer/cmd/root.go:15]: Please separate standard library imports from the other imports and perhaps please place imports into these blocks, each separated by a new line.

Block 1. Standard library imports

Block 2. Imports not from this package/library

Block 3. Imports from this package/library

**Review by odeke-em — CHANGES_REQUESTED** [review]

File: icq-relayer/cmd/root.go (hunk 1 above)
Comment (hoank101) [reply icq-relayer/cmd/root.go]: thank you for review, i just updated

**Comment by joe-bowman** [comment]
@hoank101 can you rebase on the target branch to incorporate Trung's logging changes please?

**Comment by hoank101** [comment]
> @hoank101 can you rebase on the target branch to incorporate Trung's logging changes please?

thank you, I just update

**Pull request merged**